yt_dlp/extractor/allstar.py

```diff
@@ -0,0 +1,253 @@
+import functools
+import json
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    OnDemandPagedList,
+    int_or_none,
+    join_nonempty,
+    parse_qs,
+    traverse_obj,
+    urljoin,
+)
+
+
+_FIELDS = '''
+    _id
+    clipImageThumb
```

Comment:
when I was checking out the site's API, I noticed there is also a `clipImageSource` field. have you compared the resolution/quality between the two?

Reply:
`clipImageSource` has high resolution than `clipImageThumb`
```
png 1920x1080
mjpeg 850x478
```
should we switch to `clipImageSource`?

Reply:
I think it would be better, or we could extract both as `thumbnails`

Reply:
ok done.